demo/public/css/main.css

```diff
@@ -807,7 +807,7 @@ header {
 
 .header__content .nav__link span {
   font-size: 1em;
-  font-size: 1.125em;
+  font-size: 1em;
```

Comment:
are these two lines the same? should we delete one of them?

Reply:
I'm just editing what's here... I guess they are the same? I don't know if setting multiple CSS attributes can have back-off effects or something.

Reply:
almost certainly it was a mistake in the original to set `font-size` twice, it's just more clearly one when we're setting it to the same thing both times :)